prusti-common/src/vir/ast/expr.rs

```diff
@@ -339,6 +339,18 @@ impl Expr {
         Expr::LabelledOld(label.to_string(), box expr, Position::default())
     }
 
+    pub fn const_int<I: Into<i64>>(i: I) -> Self {
+        Expr::Const(Const::Int(i.into()), Position::default())
+    }
+
+    pub fn const_bigint<S: Into<String>>(s: S) -> Self {
+        Expr::Const(Const::BigInt(s.into()), Position::default())
+    }
+
+    pub fn const_bool(b: bool) -> Self {
+        Expr::Const(Const::Bool(b), Position::default())
+    }
```

Comment:
Aren't there already `Into` implementations for (some of) these?

Reply:
hm, yes; that makes the PR one commit shorter i guess :-)